contents/java/JUnit.md

```diff
@@ -0,0 +1,226 @@
+<frontmatter>
+  title: JUnit
+  footer: footer.md
+  head: head.md
+  siteNav: mainNav.md
+  pageNav: 3
+</frontmatter>
+
+{{ navbar | safe }}
+
+<div class="website-content">
+
+# JUnit
+
+<b>Author: [Lin Si Jie](https://github.com/sijie123)</b>
+
+Reviewers: [Aadyaa Maddi](https://github.com/amad-person), [Marvin Chin](https://github.com/marvinchin)
+
+## What is JUnit?
+_____
+
+JUnit is an open-source <popover effect="fade" content="An automatic test running and verfication process to determine the correctness of a system programmatically">automated testing</popover> framework for Java. Using its <popover effect="fade" content="Application Programming Interfaces">APIs</popover>, developers can write and run tests to check for correctness in their software engineering projects. 
+
+To get a better idea of what JUnit does, let's see how JUnit works with an <popover effect="fade" content="better known system under test or SUT" placement="top">example:</popover>
+
+```java
+public class Repeater {
+  // Returns copies of reference string separated by the '\n' character.
+  public static String repeat(String reference, int copies) {
+    String result = "";
+    for (int i = 0; i <= copies; i++) {
+      result += reference + "\n";
+    }
+    return result;
+  }
+}
+```
+
+Let's write a test to check the correctness of our code. 
+
+```java
+public class RepeaterTest {
+  @Test
+  public void repeatTest() {
+    assertEquals(repeat("a", 5), "a\na\na\na\na\n");
+  }
+}
+```
+<tip-box>:bulb: By convention, we append `Test` to the name of the tested class.</tip-box>
```

Comment:
Also say a few words about the intention of the test? i.e., it verifies that when the input is `("a", 5)`, ...